helpers/lib/cdnify.js

```diff
@@ -16,38 +16,45 @@ module.exports = globals => {
         const editSessionId = siteSettings.theme_session_id;
         const cdnSettings = themeSettings.cdn;
 
-        const protocolMatch = /(.*!?:)/;
-
         if (path instanceof globals.handlebars.SafeString) {
             path = path.string;
         }
 
         if (!path) {
             return '';
         }
-
-        if (/^(?:https?:)?\/\//.test(path)) {
+        
+        if (path.startsWith('//') || path.startsWith('http')) {
             return path;
         }
 
         if (['.', '/'].includes(path[0])) {
-            path = path.replace(/^[\.\/]+/, '');
+            for (var i = 0; i < path.length; i++) {
```

Comment:
Prefer `let` over `var`

Reply:
I chose var here since let is block scoped and reinitialized at every iteration of the for loop so it will run slower.

Reply:
nvm.  Looked it up some more and I guess what I read was outdated.